package WorldRewardConfig;

message TimeMultiple
{
	required uint32 time		= 1;
	required uint32 multiple	= 2;
}

message StrongHoldResource
{
	required uint32 ironbase			= 1;
	required uint32 coinbase			= 2;
	repeated TimeMultiple timemultiple	= 3;
	repeated uint32 tech_coefficient 	= 4;
}


message WorldReward
{
	required StrongHoldResource stronghold	= 1;
}